tendrl-gluster-integration.spec

```diff
@@ -1,6 +1,6 @@
 Name: tendrl-gluster-integration
 Version: 1.4.2
-Release: 1%{?dist}
+Release: 20170802T141735.d80b2d7
```

Comment:
its not required. Its updated by mistake during my test execution. I will correct it.